node_modules/oae-core/signin/js/signin.js

```diff
@@ -0,0 +1,196 @@
+/*!
```

Comment:
I think there are a couple merge issues here, and at least one `/` somehow wound up in the inviting-guests branch.